### DISCUSSION TITLE
How to use Python API?

### DISCUSSION THREAD

**Discussion by hminle** [OpenBB-finance/OpenBB#2230] [Q&A] [answered]
Hi Open BB Team,

I installed OpenBB Terminal via conda. Now I would like to use the Python API like this, but cannot figure out how
<img width="506" alt="image" src="https://user-images.githubusercontent.com/16201681/182276720-94618440-b69d-4e46-8021-eb548e255830.png">
For example, I want to import openbb api, and generate bitcoin rainbow chart, then I try this code:
```python
from openbb_terminal import api as openbb

openbb.crypto.ov.btcrb()
```
But it fails.
Could you guys please help me?
Thanks

**Comment by DidierRLopes** [comment, marked as answer]
Hey @hminle ,

We are currently working on the API, by the end of the month we should have a strong API to show some work. You can see some work towards this on this PR: https://github.com/OpenBB-finance/OpenBBTerminal/pull/2097

In the meantime using the API is not straightforward, which is what we will focus on.

Hope this helps!

**Reply by hminle** [reply]
Thank @DidierRLopes , that's great. 
Could we install the openbb api as a Python package? Like using `pip install`, `conda install` or `poetry add`?

By the way, one thing I notice is that the btcrb function should be in Overview but actually it is in Due Diligence.

**Reply by DidierRLopes** [reply]
That's one of our goals, yes 😄 

We will go through all our commands very soon and make sure they are in the correct location. This is also why this project is still in beta phase. Thanks for pointing this out.
